### DISCUSSION TITLE
Aspiring IT professional

### DISCUSSION THREAD

**Discussion by TenzHash** [community/community#36904] [GitHub Education]
Hi! I am Tenz a freshman from Bicol University and I'm glad to be part of this community. Can I ask for tips for someone like me who's just starting at programming with C language? Every tips means so much to me.

**Comment by ossanna16** [comment]
👋 Welcome to the GitHub Community on Discussions, @Tenzaaria! Happy to have you here 🙂
To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
